cmd/sriov-network-config-daemon/start.go

```diff
@@ -102,24 +112,30 @@ func runStartCmd(cmd *cobra.Command, args []string) error {
 	var config *rest.Config
 	var err error
 
-	if os.Getenv("CLUSTER_TYPE") == utils.ClusterTypeOpenshift {
+	if vars.ClusterType == consts.ClusterTypeOpenshift {
 		kubeconfig, err := clientcmd.LoadFromFile("/host/etc/kubernetes/kubeconfig")
```

Comment:
not related to this PR, but why do we need to do this in openshift ?
in-cluster-config using service account is not enough ?

Reply:
that was a request from the security team in RedHat as a way to reduce access from the daemon to nodes that he doesn't runs on

Reply:
ack, can you add a comment ?

Reply:
done